### PR TITLE
GH-887 Google Timeline Time Parser

### PR DESCRIPTION
**Pull request by dtsnell4**
Adds a timeFormat publish parameter so the developer can specify the time/date format of the data.

Fixes GH-887

Signed-off-by: Dan Snell <Dan.Snell@lexisnexis.com>

### CODE REVIEW

**Comment by dtsnell4** [comment]
@GordonSmith @mlzummo @jbrundage Please review

**Comment by mlzummo** [comment]
looks good

File: src/google/Timeline.js [hunk 1]
```diff
@@ -28,7 +28,8 @@
     Timeline.prototype.publish("timelineGroupByRowLabel", true, "boolean", "If set to false, creates one row for every dataTable entry. The default is to collect bars with the same row label into one row.",null,{tags:["Basic"]});
     Timeline.prototype.publish("timelineShowBarLabels", true, "boolean", "If set to false, omits bar labels. The default is to show them.", null,{tags:["Basic"]});
     Timeline.prototype.publish("timelineShowRowLabels", true, "boolean", "If set to false, omits row labels. The default is to show them.",null,{tags:["Basic"]});
-    Timeline.prototype.publish("timelineSingleColor", null, "string", "Colors all bars the same. Specified as a hex value (e.g., '#8d8').",null,{tags:["Basic"]});
+    Timeline.prototype.publish("timelineSingleColor", null, "html-color", "Colors all bars the same. Specified as a hex value (e.g., '#8d8').",null,{tags:["Basic"]});
+    Timeline.prototype.publish("timeFormat", "%Y-%m-%d", "string", "Time format of the data.",null,{tags:["Basic"]});
```
Comment (GordonSmith) [inline src/google/Timeline.js]: rename to "timePattern" - I am trying to keep "format" for text we format before displaying and "pattern" as a pattern we use to "deformat" text coming from the server.

**Comment by dtsnell4** [comment]
@GordonSmith done and rebased with latest

**Pull request merged**